source/_integrations/habitica.markdown

```diff
@@ -13,20 +13,29 @@ This integration allows you to monitor and manage your Habitica profile. This in
 
 There is currently support for the following device types within Home Assistant:
 
-- Sensor - Allows you to view and monitor your player data from [Habitica](https://habitica.com/) in Home Assistant.
-
-The sensors will automatically appear, after setup the Habitica component.
-
-To use the integration you should use this example configuration:
-
-```yaml
-# Minimum viable configuration.yaml entry
-habitica:
-  - api_user: YOUR_USER_ID
-    api_key: YOUR_API_KEY
-```
-
-You can specify several users, providing `api_user` and `api_key` for each.
+- Sensor:
+  - Player data: allows you to view and monitor your player data from [Habitica](https://habitica.com/) in Home Assistant. The following sensors will be available:
+    - name
+    - hp
+    - maxHealth
+    - mp
+    - maxMP
+    - exp
+    - toNextLevel
+    - lvl
+    - gp
+    - class
```

Comment:
Can we make these more human-readable by using normal descriptions?

Reply:
Not sure if i can/should just remove/rename these, since they correspond to actual sensors created in HA. I have replaced that list with a table containing both sensor names and descriptions

Reply:
That is not guaranteed in any way, can be changed (also by the user) and Home Assistant can make different entity names in case of e.g., collisions with existing ones.

Please remove those.

Reply:
Ou, so we should keep it very generic in docs? 🤔 Kinda make sense i guess. Will remove it asap